### PR TITLE
New features

### PR DESCRIPTION
**Pull request by miiu96**
- Integrated the part where an initial nonce and an initial epoch can be provided
- Updated the version of `mx-chain-go` module

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,9 +255,13 @@ The **_[config.toml](./cmd/chainsimulator/config/config.toml)_** file:
         # round-duration-in-milliseconds parameter specifies the duration of a simulated round. The timestamp between two headers will correspond to the round duration but will not reflect real-time
         round-duration-in-milliseconds = 6000
         # rounds-per-epoch specifies the number of rounds per epoch
-        # initial-round when the chain simulator will start
-        initial-round = 1
         rounds-per-epoch = 20
+        # initial-round specifies with what round the chain simulator will start
+        initial-round = 1
```
Comment (iulianpascalau) [inline README.md:260]: why not 0?

File: cmd/chainsimulator/config/config.toml [hunk 2]
```diff
@@ -10,6 +10,10 @@
         rounds-per-epoch = 20
         # initial-round when the chain simulator will start
         initial-round = 1
```
Comment (iulianpascalau) [inline cmd/chainsimulator/config/config.toml:12]: why not 0?

**Review by sstanculeanu — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (miiu96) [reply README.md]: changed

File: cmd/chainsimulator/config/config.toml (hunk 2 above)
Comment (miiu96) [reply cmd/chainsimulator/config/config.toml]: changed

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**